bokehjs/src/lib/models/widgets/abstract_slider.ts

```diff
@@ -114,6 +114,39 @@ export abstract class AbstractSliderView extends ControlView {
 
   protected abstract _calc_from(values: number[]): number | number[]
 
+  protected abstract _set_keypress_handles(): void
+
+  protected _keypress_handle(e: KeyboardEvent, idx: 0 | 1 = 0): void {
+    const {start, value, end, step} = this._calc_to()
+    const is_range = value.length==2
+    let low = start
+    let high = end
+    if (is_range && idx==0) {
+      high = value[1]
+    } else if (is_range && idx==1) {
+      low = value[0]
+    }
+    switch (e.which) {
+      case 37: {
+        value[idx] = Math.max(value[idx] - step, low)
+        break
+      }
+      case 39: {
+        value[idx] = Math.min(value[idx] + step, high)
+        break
+      }
+      default:
+        return
+    }
+    if (is_range)
+      this.model.properties.value.change.emit()
```

Comment:
In order to trigger a sync with Python there will need to be an actual, real assignment to `this.model.value`